Dockerfiles/Dockerfile.titiler

```
@@ -5,8 +5,8 @@ WORKDIR /var/task
 
 COPY cerulean_cloud/titiler_sentinel/requirements.txt ./requirements.txt
 COPY cerulean_cloud/titiler_sentinel/handler.py handler.py
-COPY cerulean_cloud/auth.py auth.py
 COPY cerulean_cloud/titiler_sentinel/templates/ ./templates
+COPY cerulean_cloud/common/auth.py auth.py
```

@jonaraphael regarding the import error on Titiler, I reviewed the logs in AWS and saw:

```
[ERROR] Runtime.ImportModuleError: Unable to import module 'handler': No module named 'cerulean_cloud'
Traceback (most recent call last):
```

Reviewing the code in `cerulean_cloud/titiler_sentinel/handler.py`, I see that I had updated the imports as such:

```diff
- from auth import api_key_auth
+ from cerulean_cloud.common.auth import api_key_auth
```

However, I forgot to create that structure within the docker image.  This should resolve the issue:

```suggestion
COPY cerulean_cloud/__init__.py ./cerulean_cloud/
COPY cerulean_cloud/common/__init__.py cerulean_cloud/common/auth.py cerulean_cloud/common
```

---

Alternatively, we could continue to use `from auth import api_key_auth` as we had previously, however I think that it's cleaner for the internal workings of the handler to more closely mimic the code layout of the repository.